test/membrane/internal-slot-protection.spec.js

```diff
@@ -15,8 +15,6 @@ describe('membrane', () => {
             const originalProto = HTMLLinkElement.prototype;
             const link = document.createElement('link');
             link.__proto__ = HTMLElement.prototype;
-            link.setAttribute('rel', 'import');
-
             // this attack is trying to change the proto chain, and therefore the instanceof checks and such on distortions,
             // but it doesn't work because the object graph mutation for the prototype is only visible inside the sandbox.
```

Comment:
What's this change about :point_up:

Reply:
duplicate line in a test... line 20 does the same operation again.